tests/core/full_node/test_mempool.py

```diff
@@ -95,6 +101,29 @@ async def test_basic_mempool(self, two_nodes):
         assert spend_bundle is not None
 
 
+@peer_required
+@api_request
+@bytes_required
+async def respond_transaction(
```

Comment:
this is "inspired" by `FullNodeAPI.respond_transaction()`, but it preserves the return values from `FullNode.respondTransaction()`, to allow tests to assert they are what we expect them to be